themes/saleor/styles/_partial/_header.scss

```diff
@@ -1,17 +1,58 @@
 header {
-  padding-top: $global-padding*2;
-  padding-bottom: $global-padding*2;
+  .notice-bar {
+    height: 70px;
+    background: $blue;
+    color: #fff;
+    .notice-text {
+      padding-top: 26px;
```

Comment:
it'll be better to be consistent and use the same unit of measurement in the project. So use rem instead of px here and in other places if it's possible